src/messaging/ExchangeMgr.cpp

```diff
@@ -106,8 +106,13 @@ ExchangeContext * ExchangeManager::NewContext(const SessionHandle & session, Exc
 {
     if (!session->IsActiveSession())
     {
+#if CHIP_ERROR_LOGGING
         // Disallow creating exchange on an inactive session
```

Comment:
nit: maybe comment should be moved up or tied to `return nullptr` ?

Reply:
Will do.

Reply:
Done.